### ISSUE TITLE
Improve dragging item image bitmap creation process

### ISSUE THREAD

**Issue by megaswet** [h6ah4i/android-advancedrecyclerview#325]
Hi,

Thanks for default animations in 0.10.1, now is ok.

But now I have a problem with "dragging item bitmap by dragging child" fix.

Now when I hold dragging item (child item for touch dragging) I see shadow of material_shadow_z3 (I am using your shadow resource for `setDraggingItemShadowDrawable` in `RecyclerViewDragDropManager`) and shifted to left and little down Dragging bitmap. Also if I hold item0 it shifted about 30dp, if I hold item1 it shifted 90dp and so on.

**Comment by kuffs** [comment]
Confirmed. Reverted to 0.10.0

**Comment by h6ah4i** [comment]
Oops 😢 I've made a small mistake when calling `View.layout()` method. I just fixed it and released as v0.10.2. Thanks for the report!

```diff
-        v.layout(viewTop, viewLeft, viewLeft + viewWidth, viewTop + viewHeight);
+        v.layout(viewLeft, viewTop, viewLeft + viewWidth, viewTop + viewHeight);
```
